### ISSUE TITLE
how to import the library using es6

### ISSUE THREAD

**Issue by vpezeshkian** [videojs/videojs-contrib-hls#577]
So I'm using es6 and use imports. video.js itself is working but when I want to import 'videojs-contrib-hls' it doesn't see the module. 
`import vjs from 'videojs-contrib-hls'` throws error.

any suggestion?

**Comment by gkatsev** [comment]
What version are you using currently? The current versions do not allow this. But will soon be available. See https://github.com/videojs/videojs-contrib-hls/issues/490, https://github.com/videojs/videojs-contrib-hls/issues/509, and https://github.com/videojs/videojs-contrib-hls/issues/250. Thanks.

**Comment by vpezeshkian** [comment]
Version 2.0.0
Do you have ETA?

**Comment by dmlap** [comment]
When it's ready? ;) Rough guess: a week or so.

**Comment by vpezeshkian** [comment]
:+1:  
Could you please notify me on this thread? 
Thank you.

**Comment by gaglage** [comment]
solved? any news?

**Comment by forbesjo** [comment]
This should be working, here's an example
```js
var videojs = require('video.js');
require('videojs-contrib-hls');

var videoEl = document.createElement('video');
videoEl.id = 'vid';
videoEl.setAttribute('controls', '');
videoEl.setAttribute('width', '600');
videoEl.setAttribute('height', '300');
videoEl.className = 'video-js vjs-default-skin';
document.body.appendChild(videoEl);

var player = videojs('vid');

player.ready(function() {
  player.src({
    src: '//d1sh0fycs31s5a.cloudfront.net/eleph/index.m3u8',
    type: 'application/x-mpegURL'
  });
});
```

**Comment by gaglage** [comment]
Thansk for the answer.
 Im'getting an error (Uncaught TypeError: Cannot convert undefined or null to object, 	    var cacheKeys = Object.keys(cache);). Maybe webpack configuration. I'll try to find the solution! thanks

**Comment by gkatsev** [comment]
See #600 for issues with webpack and solutions around it in the meantime.

**Comment by nehakhandelwal** [comment]
I too facing the same issue .

**Comment by AdamTyler** [comment]
@gaglage did you find a workaround for your issue? I'm seeing the same error

**Comment by gaglage** [comment]
Hi @AdamTyler I fixed the issue via #600
